### ISSUE TITLE
checkpoints 

### ISSUE THREAD

**Issue by JingChenJXC** [Wi-sc/NVF#1]
Hi, congrats on the great work. I was wondering, do you plan to release checkpoints for some categories in ShapeNet?

**Comment by Wi-sc** [comment]
Thanks for your interest.  The model of non-watertight shapes and garments has been uploaded. You can find the download link in the readme. 

The model was further refined and should perform better than the original model reported in the paper. Please feel free to try it.

**Comment by JingChenJXC** [comment]
Thank you very much for sharing the pretrained models. This is a significant time-saver for me. I've just submitted the request for access. May I inquire how you refined the model?

**Comment by Wi-sc** [comment]
Of course. The new model was trained for a longer time.

**Comment by JingChenJXC** [comment]
No offense. Since your name looks like a Chinese name too, may I ask if you
speak Chinese?  If yes, do you mind to communicate in Chinese?

Best wishes,
Jing Chen(陈静)

Electrical Engineering & Computer Science Dept.

Case Western Reserve University

Cleveland, OH 44106


Office: Olin 513
Phone: 1-216-280-3439
Email: ***@***.***

**Comment by Wi-sc** [comment]
Hi Jing,


Sorry for the late reply. I didn't check this email in the last few weeks.


Yes, we can talk in Chinese. Just let me know your questions.


Best,

Xianghui













At 2023-08-30 03:46:48, "JingChenJXC" ***@***.***> wrote:

No offense. Since your name looks like a Chinese name too, may I ask if you
speak Chinese? If yes, do you mind to communicate in Chinese?

Best wishes,
Jing Chen(陈静)

Electrical Engineering & Computer Science Dept.

Case Western Reserve University

Cleveland, OH 44106


Office: Olin 513
Phone: 1-216-280-3439
Email: ***@***.***

**Comment by JingChenJXC** [comment]
Xianghui，
您好！

不好意思，我刚发现您给我回复啦！能收到您的回信，还可以切中文交流，真的是太开心了。我最近正在将我们的模型和N
VF进行对比。做实验时，既想保持方法对比的公平性，
又想突出自己方法的特点，导致许多实验细节无法定案。能和您取得联系，真的是很惊喜。我就不客气了，直接交流了哈！

1）首先是neural vector field这个术语。我现在沿用了您这个术语，可老实说，我总觉得这个
术语有些宽泛了。不知道您们当时为什么何定下这个术语？从数学上说：任何一个$R^n-> R^m$ mapping 都可以被称为vector
field。我也想过用displacement或者叫gradient field。但是，displacement
已经被用在了SDF相关的研究里，它们借鉴computer graphics里的displacement map。vector
field严格上说，也不是UDF的gradient field，而是0.5 * UDF^2的 gradient field。

2）在您工作的基础上，我们设计开发了一个模型，该模型的输出的也是vector
field，理论上，模型预测误差能更直接地体现模型准确性。但不知道这样比较，对于NVF是否公平？

3）我自己读文章的理解是：用近邻点特征来计算查询点特征，好处是模型的泛化能力较强，但包含一个implicit
assumption：输入点的密度比较均匀，且training和testing的点密度比较接近。不过这个想法一直都停留在主观臆断，没实际做过实验。不知道您们是否测在sparse，noisy，或
non-uniform的点集合上测试过NVF？效果如何？

祝好，
陈静

Electrical Engineering & Computer Science Dept.

Case Western Reserve University

Cleveland, OH 44106


Office: Olin 513
Phone: 1-216-280-3439
Email: ***@***.***
